README.md

```diff
@@ -322,21 +322,14 @@ You can render the HTML yourself with ReDoc:
 
 ## Updating from 1.x
 
-* `header.md` change to `body.json`
-    ```json
-      {
-        "openapi": "3.0.0",
-        "info": {
-          "title": "Dox Example",
-          "description": "description.md",
-          "contact": {},
-          "version": "2.0"
-        }
-      }
-    ```
-* update rspec config file
-  * `header_file_path` -> `body_file_path`
-* Add needed schema paths
+* remove depricated configs
```

Comment:
I'd make this more readable:

BREAKING CHANGES:
removed configs:

renamed configs:

NON-BREAKING CHANGES (unless these are also breaking:
new (optional) configs:

PS: deprecated != removed code

Reply:
It still isn't removed code because I wanted it to work with the same config as before.

Basically there are not breaking changes :D

Reply:
Sorry, that's cool!